base_app/yelp_service.py

```diff
@@ -23,13 +24,14 @@ def get_restaurant(self, params):
         url = 'https://api.yelp.com/v3/businesses/search'
         response = self.connection(url, params)
         all = response.json()
+        ids = {}
         for x in all['businesses']:
             if x.get('price') != None:
-                ids = []
-                ids.append(x['id'])
+                ids.update({x['id']: x['distance']})
```

Comment:
Can you briefly explain what this line is doing? Why `distance`?

Reply:
Thanks for catching that! I've taken it out.